core/ledger/kvledger/kv_ledger_provider_test.go

```diff
@@ -214,7 +214,7 @@ func TestLedgerBackup(t *testing.T) {
 	provider.Close()
 
 	// Create restore environment
-	env := createTestEnv(t, restorePath)
+	env := createTestEnvWithoutSetupExtTestEnv(t, restorePath)
```

Comment:
Perhaps some context on why this change is needed would help.

Reply:
this is needed for "Initializing ledger provider", otherwise kvledger.NewProvider() will get stuck waiting for couchdb connection